### ISSUE TITLE
ifdb chat

### ISSUE THREAD

**Issue by qdacsvx** [iftechfoundation/ifdb-suggestion-tracker#142]
There should be a live chat box. Which pages should have a chat box?

Would people be able to chat anonymously (i.e. without logging in)?

**Comment by curiousdannii** [comment]
This is getting way outside the current purpose of the IFDB. The IFDB is barely even in the sphere of a social network. Everyone is free to chat in Discord etc. if they want. Moderating a chat box, even if it wasn't anonymous, is surely not what any of the moderators signed on for.

**Comment by qdacsvx** [comment]
@curiousdannii I 100% support the IFDB moderation team. While I could ask for a show of hands for moderators to volunteer. If you tell me no one should be subjected to this I will accept closing it.

While I could rein back from a wide open chat box, I would still want say one moderator and one logged-in user to chat on-demand to achieve real time help. I understand if that is simply too much to ask for. Moderators are volunteers giving their free time. I have the greatest respect for every one of the IFDB moderators who have an essential role in making a better IFDB.

I've been thinking about features to help people use IFDB particularly newbies. But there are other ways to do this.

**Comment by Durafen** [comment]
I personally like the idea of an IF chat, a more fast way to reach out and get fast feedback.
I do however understand the complexity and the requirements for moderation, pros and cons...

**Comment by dfabulich** [comment]
I think we wouldn't/shouldn't write a chat system ourselves. Instead, we'd add some links from IFDB to an existing chat system (Discord, ifMUD, IRC, whatever) that was governed by IFTF.

I claim that no such space currently exists. NarraScope has a Discord, but it isn't/wasn't intended to be a year-round watering hole. It's pretty quiet right now. ifMUD has users year-round, but they seem pretty happy with what they've got, and are not interested in expansion, or writing software to make it easier to use, or what have you.

So, in practice, the first step would be for a volunteer to build out a chat space and recruit some users, e.g. on Discord. Then, we could talk about adding links to it, when it gets big enough.

**Comment by NilsFagerburg** [comment]
Wouldn't this rather be something for the IntFiction Forum Committee?

**Comment by curiousdannii** [comment]
@NilsFagerburg Haha, take my comment as indicative of what the forum committee would say. We're staying out of it! If the IFDB committee wants to do something they're free to do so.

**Comment by dfabulich** [comment]
Yeah, to be clear, IFTF doesn't have a ton of spare time waiting around to kick off big volunteer projects.

Thus, the way forward is clear: if somebody wants to do this, they should create a chat space (most likely on Discord, but hey, it's your show), recruit some users, and then ask us to add links to it.

**Comment by dougo** [comment]
We could at least have a link to the existing forum.

**Comment by Oreolek** [comment]
IFDB already has game comments. IFMud is realtime and has guest access. Both aren't exactly bursting with activity.
